SingularityUI/app/components/common/RemoveButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component, PropTypes } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+
+export default class RemoveButton extends Component {
```

Comment:
`<Button className='remove-button'` isn't many more characters than `<RemoveButton`, but adds the burden of being yet another component to be aware of -- is it really necessary?